index.html

```diff
@@ -83,5 +83,270 @@ <h1 class="text-dark hero-title">
+            </div>
+          </div>
+        </div>
+        <div class="box flex flex-col">
+          <div class="box-img">
+            <img
+              class="w-full h-full"
+              src="./assets/images/img-placeholder.png"
+              alt="imgPlaceholder"
+            />
+          </div>
+          <div class="box-content fw-600">
+            <h2 class="">Multi-Post Stories Gain + Glory</h2>
+            <div class="tags flex w-full justify-between">
+              <span>Ruby on Rails</span>
+              <span>css</span>
+              <span>JavaScript</span>
+              <span>HTML</span>
+            </div>
+            <div class="flex justify-center">
+              <button
+                class="text-center btn text-white"
+                type="button"
+              >
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="box flex flex-col">
+          <div class="box-img">
+            <img
+              class="w-full h-full"
+              src="./assets/images/img-placeholder.png"
+              alt="imgPlaceholder"
+            />
+          </div>
+          <div class="box-content fw-600">
+            <h2 class="">Multi-Post Stories Gain + Glory</h2>
+            <div class="tags flex w-full justify-between">
+              <span>Ruby on Rails</span>
+              <span>css</span>
+              <span>JavaScript</span>
+              <span>HTML</span>
+            </div>
+            <div class="flex justify-center">
+              <button
+                class="text-center btn text-white"
+                type="button"
+              >
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="box flex flex-col">
+          <div class="box-img">
+            <img
+              class="w-full h-full"
+              src="./assets/images/img-placeholder.png"
+              alt="imgPlaceholder"
+            />
+          </div>
+          <div class="box-content fw-600">
+            <h2 class="">Multi-Post Stories Gain + Glory</h2>
+            <div class="tags flex w-full justify-between">
+              <span>Ruby on Rails</span>
+              <span>css</span>
+              <span>JavaScript</span>
+              <span>HTML</span>
+            </div>
+            <div class="flex justify-center">
+              <button
+                class="text-center btn text-white"
+                type="button"
+              >
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section relative flex flex-col align-items-center">
+      <div class="about-bg"></div>
+      <div class="about-container">
+        <header class="about-content fw-700 leading-4 w-full text-center mb-2">
+          <h1 class="text-dark about-title">About Me</h1>
+        </header>
```

Comment:
- [ ] Why are you using the `header` tag here :smile: please remember that the `header` typically contains the logo and the navigations so it would be better if you consider using `div` :+1: